src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: 10px;
+  width: $calendar-width;
```

Comment:
The variable `$calendar-width` is used here but is not defined within this file. Ensure that it is defined or imported from another file to avoid compilation errors.